packages/shared/.eslintrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+    rules: {
+        'import/no-unresolved': 'off',
```

Comment:
same here, any reason for disabling this?

Reply:
configuration removed